server/neural_network_interface.py

```diff
@@ -1,18 +1,121 @@
+from keras.models import load_model
+import numpy as np
+from itertools import chain
+
 _movement_count_dict = {}
 
-def generate_move(sid, gesture_data, message_data):
-    """Returns the (position data, movement count) that will be sent to the
-    client.
+"""
+~~~~~ This code was copied and pasted from the neural net repo. It should
+      be factored out in both places. ~~~~~
+"""
```

Comment:
It would probably make sense to load in the Neural Network repo as a git submodule inside the server rather than copy-paste in. The interface would import the necessary modules, and `NeuralNetworkBlocksworldModel` would call the necessary functions to set everything up and process moves.

We can handle that in a later commit, though.